src/app/utils/utils.js

```diff
@@ -747,6 +747,14 @@ function isNyplBnumber(bnum) {
   return /^b/.test(bnum);
 }
 
+/**
+ * Strip check digit (int at 10th position) from bnumber if present
+ * Return the original bnumber if greater than or less than 10 characters.
+ */
+function removeCheckDigit(bnum) {
```

Comment:
Can you assigned it a default value? Such as `(bnum = "")`. Don't think we'll ever run into this case but as precaution.